_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
```

Comment:
I had to read Understanding the SC to understand what this rule is trying to do. It think the description could be a little clearer. Perhaps something like:

```suggestion
  This rule checks that text spacing set in the `style` attribute is not set to `!important` or that particular text styles have exceeded the specified metric for retaining content visibility and functionality.
```